Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
   "client",
   "cli",
   "portal",
+  "oo7-daemon",
```

Comment:
I meant renaming the project in Cargo.toml, not the directory itself. The name was good there.

Reply:
Done.